### PR TITLE
docs(components): update docs

### PR DESCRIPTION
**Pull request by emojiiii**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: docs/en-US/component/date-picker.md [hunk 1]
```diff
@@ -165,6 +165,7 @@ Note, date time locale (month name, first day of the week ...) are also configur
...
 | disabledDate          | a function determining if a date is disabled with that date as its parameter. Should return a Boolean | function                                         | —                                                                                                                 | —           |
 | shortcuts             | an object array to set shortcut options                                                               | object[{ text: string, value: date / function }] | —                                                                                                                 | —           |
+| cellClassName         | set custom className                                                                                  | Function(Date)                                   | —                                                                                                                 | —           |
```
Comment (sxzz) [inline docs/en-US/component/date-picker.md:168]:
```suggestion
| cell-class-name         | set custom className                                                                                  | Function(Date)                                   | —                                                                                                                 | —           |
```

File: docs/en-US/component/date-picker.md [hunk 2]
```diff
@@ -165,6 +165,7 @@ Note, date time locale (month name, first day of the week ...) are also configur
 | validate-event        | whether to trigger form validation                                                                    | boolean                                          | -                                                                                                                 | true        |
 | disabledDate          | a function determining if a date is disabled with that date as its parameter. Should return a Boolean | function                                         | —                                                                                                                 | —           |
```
Comment (sxzz) [inline docs/en-US/component/date-picker.md:166]:
```suggestion
| disabled-date          | a function determining if a date is disabled with that date as its parameter. Should return a Boolean | function                                         | —                                                                                                                 | —           |
```

Comment (JeremyWuuuuu) [reply docs/en-US/component/date-picker.md:166]: Agreed

File: docs/en-US/component/steps.md [hunk 3]
```diff
@@ -97,12 +97,12 @@ steps/simple
...
-| icon        | step icon's class name. Icons can be passed via named slot as well       | string | —                                         | —       |
-| status      | current status. It will be automatically set by Steps if not configured. | string | wait / process / finish / error / success | —       |
+| Attribute   | Description                                                              | Type             | Accepted Values                           | Default |
+| ----------- | ------------------------------------------------------------------------ | ---------------- | ----------------------------------------- | ------- |
+| title       | step title                                                               | string           | —                                         | —       |
+| description | step description                                                         | string           | —                                         | —       |
+| icon        | step custom icon. Icons can be passed via named slot as well             | string/Component | —                                         | —       |
```
Comment (JeremyWuuuuu) [inline docs/en-US/component/steps.md:104]: Can we insert a space like:

`string / Component`?

**Review by jw-foss — COMMENTED** [review]
A few comments added, PTAL.

File: docs/en-US/component/steps.md (hunk 3 above)
Comment (emojiiii) [reply docs/en-US/component/steps.md:104]: ok!

File: docs/en-US/component/date-picker.md (hunk 2 above)
Comment (emojiiii) [reply docs/en-US/component/date-picker.md:166]: ok!

**Review by sxzz — APPROVED** [review]

**Pull request merged**